docs/Server-Setup.md

```diff
@@ -138,8 +138,10 @@ hour because lots of packages have to be installed):
 ## Testing
 
 Since tests run against an existing instance, make sure you update your instance first.
+You also need to set the CODALAB_LINK_MOUNTS environment variable so that the `link` tests
+can properly succeed.
 
-    ./codalab_service.py start -bd -s rest-server
+    CODALAB_LINK_MOUNTS=/tmp ./codalab_service.py start -bd -s rest-server
```

Comment:
Can we make this default to `var/codalab/link-mounts` or something?
1) /tmp might have stuff from other applications we don't want to expose by default
2) It'd be nicer to have a default so we don't have to worry about setting the environment variable to get sane behavior (e.g., for running tests).

Reply:
Done